Tomas Heran <tomas.heran@oracle.com>
Relax the gettext version requirement. Will not be reported upstream.

diff -r 4efb170d3a5e -r 8da9a2931d0d configure.ac
--- a/configure.ac	Fri Feb 03 14:25:22 2017 +0100
+++ b/configure.ac	Fri Feb 03 16:01:35 2017 +0100
@@ -36,7 +36,7 @@
 
 # check for gettext
 AM_GNU_GETTEXT([external])
-AM_GNU_GETTEXT_VERSION([0.19.8])
+AM_GNU_GETTEXT_VERSION([0.19])
 
 AC_MSG_CHECKING([[whether z/OS special settings are required]])
 if test `uname -s` = "OS/390"
